frontend/src/app/api/search/export/route.ts

```diff
@@ -0,0 +1,34 @@
+import { downloadOpportunities } from "src/services/fetch/fetchers/searchFetcher";
+import { convertSearchParamsToProperTypes } from "src/utils/search/convertSearchParamsToProperTypes";
+
+import { NextRequest, NextResponse } from "next/server";
+
+export const revalidate = 0;
+
+/*
+  so the request flow here goes something like:
+
+  ExportSearchResultsButton click ->
+  /export route ->
+  downloadOpportunities ->
+  fetchOpportunitySearch ->
+  ExportSearchResultsButton (handle response by blobbing it to the location) -> user's file system
+
+*/
+
+export async function GET(request: NextRequest) {
```

Comment:
making API request here in order to avoid exposing the API key

Reply:
Seems like a shame we have to do this. Maybe at some point we can open up some of the APIs for the general public. 

Noting the API is a `POST` while we're using a `GET`, I think `GET` is the correct interpretation for search since a search request isn't meaningfully creating or updating server entities.

Reply:
Agree it's not great, but the best option we have for now. We should maybe talk as a team about how we want to handle this sort of thing.

As far as the GET v POST, I'd be fine to make the first request as POST as well, but since it's largely internal and not sending a request body, a GET seemed fine. The search itself being a post makes sense to me since we're sending data in the body of the request.